tests/helper.py

```diff
@@ -0,0 +1,25 @@
+import re
+
+import labgrid.protocol
+
+
+class SystemdRun:
```

Comment:
We should add a comment like:
`# work around until https://github.com/labgrid-project/labgrid/pull/835 is merged`

Reply:
Done. And also updated the documentation for the class as a whole.